tests/lax_numpy_test.py

```diff
@@ -238,6 +238,8 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
     op_record("remainder", 2, default_dtypes, all_shapes, jtu.rand_nonzero, [],
               tolerance={onp.float16: 1e-2}),
     op_record("mod", 2, default_dtypes, all_shapes, jtu.rand_nonzero, []),
+    op_record("rint", 1, all_dtypes + uint_dtypes, all_shapes,
+              jtu.rand_some_inf_and_nan, []),
```

Comment:
I also wonder if it's valuable testing large floats?

Reply:
We don't do that for floor or ceil, so why for rint? Also is there a straightforward way to do that?

Reply:
The reason I'm wondering about that specifically is that I'm wondering what happens to the odd/even test when you are outside the range of floats where you can represent all integers.

I think the easiest way to do this would be to add another RNG to `jax.test_util` (or see if another of the ones there does something useful).

Reply:
Now that I'm using round_to_nearest_even is this a moot point? FWIW the version I had implemented was based on the implementation I found in Numpy https://github.com/numpy/numpy/blob/31ffdecf07d18ed4dbb66b171cb0f998d4b190fa/numpy/core/src/npymath/npy_math_internal.h.src#L279-L298.

Reply:
It's fine for now. I still wonder if we are correct though.